handbook/company.md

```diff
@@ -449,3 +449,9 @@ The following Slack channels are maintained by Fleet's founders and executive co
 
 
 <meta name="maintainedBy" value="mikermcneil">
+<!-- 
+Add pageOrderInSection meta tags to each handbook page
+<meta name="pageOrderInSection" value="100">
```

Comment:
Could probably get away starting without `pageOrderInSection` and just doing alphabetical.